tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -901,7 +901,7 @@ export class WebGPUBackend extends KernelBackend {
   private recordAndSubmit(
       program: webgpu_program.WebGPUProgram, output: TensorInfo,
       inputs: TensorInfo[], programDefinedUniform?: ProgramUniform) {
-    if (program.pipeline instanceof Promise<GPUComputePipeline>) {
```

Comment:
This is not meaningful in TypeScript because TS fully erases types. Checking `instanceof Promise` is the best we can do at runtime.